manager/state/raft/raft.go

```diff
@@ -542,6 +542,9 @@ func (n *Node) Run(ctx context.Context) error {
 		n.done()
 	}()
 
+	// Flag to indicate if this manager node was the raft leader in the previous iteration of the loop.
+	// It is mainly used to cancel out proposals that this node might have made when it was the leader
+	// in the previous iteration. It not only needed to avoid deadlocking processCommitted().
```

Comment:
I don't quite understand the last sentence - seems like it's missing a word.

Reply:
I can't understand it either. Don't remember what I was trying to say. I'll fix/remove it.